azure-pipelines.yml

```diff
@@ -8,21 +8,21 @@ trigger:
 
 strategy:
   matrix:
-    linux_node_8:
-      imageName: 'ubuntu-16.04'
-      node_version: 8.x
     linux_node_10:
-      imageName: 'ubuntu-16.04'
+      imageName: 'ubuntu-18.04'
       node_version: 10.x
-    linux_node_latest:
-      imageName: 'ubuntu-16.04'
+    linux_node_lts:
+      imageName: 'ubuntu-18.04'
       node_version: 12.x
+    linux_node_latest:
+      imageName: 'ubuntu-18.04'
+      node_version: 13.x
     mac_node_latest:
-      imageName: 'macos-10.13'
-      node_version: 12.x
+      imageName: 'macOS-10.15'
+      node_version: 13.x
     windows_node_latest:
-      imageName: 'vs2017-win2016'
-      node_version: 12.x
+      imageName: 'windows-2019'
+      node_version: 13.x
```

Comment:
Shouldn't we rather use LTS on mac / windows?

Reply:
Hm, isn't latest more probable to show different behavior on different systems?

Reply:
Yeah I think you are right.